src/main/java/com/sjsu/backbenchers/vBless/controller/CampaignRestController.java

```diff
@@ -76,7 +76,7 @@
 	public ResponseEntity<Campaign> createCampaign(@RequestBody Campaign campaign){
 		log.debug("createCampaign");
 		System.out.println("createCampaign");
-		campaign.setActive("Y");
+		campaign.setStatus("active");
```

Comment:
can we make it like 'A' for active, 'I' for inactive etc in all places. We will get the name from lookup table as we discussed in last meeting.